spec/features/destroy_deck_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe 'deck/destroy' do
+  let!(:player) { Player.create(name: 'Karl') }
+  let!(:deck) { Deck.create(name: 'Llanowar Elf', player: player) }
+
+  before do
+    visit players_path
+    click_on(player.name.to_s)
+    click_on(deck.name.to_s)
```

Comment:
just like the `visit` call it's usual to have no parenthesis when using a DSL in ruby:
```rb
click_on player.name
```
pretty sure you don't need `.to_s` also.

Reply:
I would do that for every `click_on` call unless you want to disambiguate some method chaining.

Reply:
you just need to be consistent here, for example `visit` you didn't add parenthesis.

Reply:
done I changed all the instances in the feature specs